### PR TITLE
ADD acceptance tests for `ec validate input`

### PR DESCRIPTION
**Pull request by robnester-rh**
This commit adds acceptance tests for `ec validate input`.

Resolves: EC-364

### CODE REVIEW

**Comment by lcarva** [comment]
@robnester-rh, why was this merged without a review?

File: features/__snapshots__/validate_input.snap [hunk 1]
```diff
@@ -0,0 +1,55 @@
+
+[invalid policy URL:stdout - 1]
```
Comment (lcarva) [inline features/__snapshots__/validate_input.snap:2]: I don't see a scenario for this. Outdated snapshot?

**Comment by robnester-rh** [comment]
> @robnester-rh, why was this merged without a review?
Apologies, I'd worked with @simonbaird on these directly. I'll make sure there's at least one lgtm next time.

File: features/__snapshots__/validate_input.snap (hunk 1 above)
Comment (robnester-rh) [reply features/__snapshots__/validate_input.snap:2]: Yes. I'll update and create a PR.

**Pull request merged**